### PR TITLE
Add suggestion for contributing with documentation

### PR DESCRIPTION
**Pull request by lilianabs**
Adding a small text with some guidelines on how to add documentation to existing models.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,28 @@ Each model can be used in [Gitpod](https://www.gitpod.io/), just [open the repos
...
 
-We welcome contributions of new models. They should be in a folder with a project and manifest file, to pin all relevant packages, as well as a README to explain what the model is about, how to run it, and what results it achieves (if applicable). If possible models should not depend directly on GPU functionality, but ideally should be CPU/GPU agnostic. Please keep the code short, clean and self-explanatory, with as little boilerplate code as possible.
+We welcome contributions of new models and documentation. 
+
+### Share a new model
+
+If you want to share a new model, then we suggest you follow these guidelines:
+
+* Models should be in a folder with a project and manifest file to pin all relevant packages.
+* Models should include README to explain what the model is about, how to run it, and what results it achieves (if applicable).
```
Comment (ToucheSir) [inline README.md:36]:
```suggestion
* Models should include a README(.md) to explain what the model is about, how to run it, and what results it achieves (if applicable).
```

File: README.md [hunk 2]
```diff
@@ -26,7 +26,28 @@ Each model can be used in [Gitpod](https://www.gitpod.io/), just [open the repos
...
+* If possible models should not depend directly on GPU functionality, but ideally should be CPU/GPU agnostic.
```
Comment (ToucheSir) [inline README.md:37]:
```suggestion
* Models should ideally be CPU/GPU agnostic and not depend directly on GPU functionality.
```

File: README.md [hunk 3]
```diff
@@ -26,7 +26,28 @@ Each model can be used in [Gitpod](https://www.gitpod.io/), just [open the repos
...
+* Please keep the code short, clean, and self-explanatory, with as little boilerplate code as possible.
```
Comment (ToucheSir) [inline README.md:38]:
```suggestion
* Please keep the code short, clean, and self-explanatory, with as little boilerplate as possible.
```

**Review by ToucheSir — COMMENTED** [review]
Some tiny tweaks for your consideration, but otherwise LGTM

**Comment by lilianabs** [comment]
Thank you for the suggestions @ToucheSir! :)

**Review by ToucheSir — APPROVED** [review]
Great!

**Pull request merged**